hangman.rb

```diff
@@ -0,0 +1,13 @@
+def hangman(word, guessed_letters)
+    result = word.chars.map do |letter|
+      guessed_letters.include?(letter) ? letter : '_'
+    end
+  
+    return result.join
```

Comment:
A common Ruby-ism is to omit the `return` on the last line because Ruby methods already return the last expression executed.

```suggestion
    result.join
```